examples/c/.gdb_history

```diff
@@ -0,0 +1,139 @@
+quit
+b 159
+c
+start
+c
+n
+start
+c
+s
+n
+n
+n
+start
+c
+s
+n
+n
+s
+n
+n
+n
+n
+n
+n
+n
+n
+n
+n
+n
+n
+quit
+b 101
+c
+start
+c
+n
+start
+c
+s
+n
+n
+n
+start
+c
+s
+n
+s
+n
+n
+n
+n
+n
+n
+n
+n
+n
+n
+start
+c
+s
+start
+c
+s
+n
+s
+n
+p solver->fcn_ptrs->copy_args
+s
+n
+n
+n
+n
+n
+n
+n
+print ns
+n
+b sim_erk_integrator.c:93
+start
+c
+c
+n
+print ns*ns*sizeof(double)
+print ns*ns
+source->A_mat[0]
+print source->A_mat[0]
+print source->A_mat[2]
+print dest->A_mat[2]
+print dest->A_mat[3]
+quit
+b 101
+c
+start
+c
+p erk_opts->A_mat
+p erk_opts->A_mat[0]
+p erk_opts->A_mat[1]
+p erk_opts->A_mat[2]
+s
+n
+n
+s
+n
+n
+n
+n
+n
+n
+n
+n
+n
+s
+n
+n
+n
+p source->A_mat
+b sim_erk_integrator.c:82
+start
+c
+c
+p source->A_mat
+b sim.c:164
+start
+c
+c
+print (sim_rk_opts *)args_->A_mat
+quit
+b sim.c:168
+start
+c
+p args->A_mat
+n
+p args->A_mat
+p args_
+p c_ptr
+s
+print source_
+print source_->A_mat
+quit
```

Comment:
@nielsvd, I thought we had completely removed qpDUNES-dev submodule from all possible places. Where did this come from?

Reply:
@dkouzoup, it was still in my ``.git/index``-file and the ``external/qpDUNES-dev``-directory still existed. I therefore got errors when executing the ``git submodule update`` command. I had to do a ``git rm ..`` to make this error go away... I suppose it has *something* to do with the fact that my local git-repository was from before the ``qpDUNES-dev``-submodule was removed.

The ``.gitmodules`` file you link to is untouched:
https://github.com/acados/acados/blob/new_core/.gitmodules
https://github.com/nielsvd/acados/blob/new_core_copy_args/.gitmodules

Reply:
Yes, I had a quite similar mess when I removed it at some point, that's why I am asking. Actually, no submodules coming after could get updated because of this error. But I thought we had eliminated it from anywhere, so perhaps, it's just because you didn't have a fresh clone. Never mind ;)